routes/services/email_service.js

```diff
@@ -29,7 +29,7 @@ exports.sendEmail = function(emailDetail, callback){
                     Please do not reply to this email.`,
             attachments: [{
                 filename: 'icon.jpg',
-                path: '/app/public/img/icon.jpg',
+                path: './public/img/icon.jpg',

```

Comment:
I would say no for this one since this works on our server (on server it requires `/app`)